### PR TITLE
update project profile of home unite us page

### LINKED ISSUE #3084: Update Project Profile: Home Unite Us
### Overview
As a product manager of Home Unite Us I want my project's details to be up to date on the HackforLA website

### Project card
https://github.com/hackforla/website/blob/gh-pages/_projects/home-unite-us.md

### Action Items

- [x] Developer: Update the [Home Unite Us project info page](https://www.hackforla.org/projects/home-unite-us) Team Leadership as requested by the PM:
   - [x] PM: Please check off everything that is ok the way that it is, and if not, please provide the updated information in the comments below:
      - [x] title
      - [x] description
      - [x] image
      - [x] leadership
         - [x] Remove Abiha Ali
         - [x] Add Ben Ross
              - [x] GitHub handle: @rossbb
              - [x] Role: Product Manager
              - [x] Slack ID: U03BZ5WL8GY
         - [x] Name
         - [x] GitHub handle
         - [x] Role
         - [x] SLACK ID
      - [x] links
         - [x] Github
         - [x] Slack
         - [x] Test Site
         - [x] Demo Site
         - [x] Site
      - [x] technologies
      - [x] partner
      - [x] tools
      - [x] status
 - [x] If you need updates to your meeting times, open an issue on VRMS to update the meeting times and provide the link to the new issue in the comment below: https://github.com/hackforla/VRMS/issues/1126

### Resources
- [VRMS Add/edit team meeting time template issue](https://github.com/hackforla/VRMS/issues/new?assignees=&labels=1+week+change+request&template=add-edit-team-meeting-time.md&title=Meeting+time+change+request+for+%5BProject+Name%5D)
- https://github.com/hackforla/website/blob/gh-pages/_projects/home-unite-us.md
- [Home Unite Us project info page](https://www.hackforla.org/projects/home-unite-us)

### PR DESCRIPTION
**Pull request by trishajjohnson**
Fixes #3084

### What changes did you make and why did you make them ?

  -Removed Abiha Ali from leadership roster
  -Added Ben Ross to leadership roster with their name, github handle, role and slack id.

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

image

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
image

</details>

### CODE REVIEW

**Comment by Jaretzbalba** [comment]
ETA: End of the day on 5/22
Availability: 2 hours

**Comment by Jaretzbalba** [comment]
Hey @trishajjohnson, can you make sure to add this pr to the project board?

**Review by Jaretzbalba — CHANGES_REQUESTED** [review]
@trishajjohnson Thank you for working on this issue! Everything looks good but for consistency among the project pages, could you please change the roles from "Product Management" to "Product Manager" for Regan, Kristy, and Ben?

**Comment by answebdev** [comment]
ETA: 5/20/2022 (end of day)
Availability: 2 hours

**Review by answebdev — CHANGES_REQUESTED** [review]
@trishajjohnson Thank you for submitting your PR. Good job removing Abiha Ali from the leadership roster and adding Ben Ross to the leadership roster. I was able to see this change locally, as well. However, the link for Ben Ross's picture used in line 51 of `_projects/home-unite-us.md` is the same link used for Abiha Ali, so this needs to be changed so that the link for Ben Ross's picture is used.

I also second what @Jaretzbalba said about changing the roles from "Product Management" to "Product Manager" for Regan, Kristy, and Ben. That way, it will be consistent with the other project pages, as in the following examples:

<details>
  <summary>Food Oasis Project Page</summary>

food_oasis

</details>

<details>
  <summary>Brigade Organizers Playbook Project Page</summary>

brigade_organizers_playbook

</details>

<details>
  <summary>100 Automations Project Page</summary>

100_automations

</details>

And also, yes, make sure to add this PR to the project board. Thank you.

**Review by blulady — DISMISSED** [review]
This pr is on the correct branch. There is a correct linked issue. The pr addresses the linked issue, namely to remove Abiha Ali from the Home Unit Us project info page and add Ben Ross. The pictures of visual changes look good. The links work in browser. The picture for Ben Ross looks like it is still Abiha. Make that Change and I think you are good to go.

**Comment by trishajjohnson** [comment]
@answebdev and @Jaretzbalba I made the changed all Product Management roles to Product Manager, but I am unsure about the picture for Ben.  In the actual issue, there was no pic given to use for Ben.  I assumed that the pic for Abiha was a generic default pic, which is why I put it there.  Did I miss the url for Ben completely or is it not there in the issue?

UPDATE:

I just realized that the pictures used on the site are their github profile pictures, so I'll get Ben's from his github.

**Comment by answebdev** [comment]
@trishajjohnson Thank you for making the change from "Product Management" to "Product Manager". But, were you able to see this change locally? I am not able to see this change. As you can see, I still see "Product Management", unless it's something on my end (see below). @Jaretzbalba Are you able to see the change to "Product Manager" locally on your end? Also, @trishajjohnson Were you able to get the link for Ben's GitHub picture?

<details>
  <summary>Home Unite Us - local</summary>

home_unite_us

</details>

**Comment by trishajjohnson** [comment]
@answebdev Sorry for the delay.  I hadn't pushed the revisions.  They are there now, so you should be able to see the changes.  I changed Product Management roles to Product Manager and I replaced Ben's picture with the correct url.  Ready to re-review.

**Review by answebdev — DISMISSED** [review]
@trishajjohnson Thank you for pushing your revisions. I had a feeling my not being able to see your changes may have had something to do with that. But now I am able to see the changes. I can now see the change from "Product Management" to "Product Manager" for Regan, Kristy, and Ben, and I am now able to see Ben's GitHub picture as well. Great job!

**Comment by kathrynsilvaconway** [comment]
ETA: End of Day on 5/30
Availability: 1 hour

**Comment by trishajjohnson** [comment]
@JessicaLucindaCheng is this ready for merging, or do I need to request for another re-review?

**Comment by JessicaLucindaCheng** [comment]
@hackforla/website-merge Please respond to @trishajjohnson: 
> is this ready for merging, or do I need to request for another re-review?

**Comment by JessicaLucindaCheng** [comment]
@trishajjohnson Please reach out to Tamara or one of the merge team members. You can find their Slack info here: https://github.com/hackforla/website/wiki/Meet-the-Team

File: _projects/home-unite-us.md [hunk 1]
```diff
@@ -49,7 +43,12 @@ leadership:
       slack: "https://hackforla.slack.com/team/U02JQLQ6YGY"
       github: "https://github.com/rpbracker"
     picture: https://avatars.githubusercontent.com/rpbracker
-
+  - name: Ben Ross
+    role: Product Manager
+    links:
+      slack: "https://hackforla.slack.com/team/U03BZ5WL8GY"
+      github: "https://github.com/rossbb"
+    picture: https://avatars.githubusercontent.com/u/5248025?v=4
```
Comment (Jaretzbalba) [inline _projects/home-unite-us.md]: @trishajjohnson To keep the code base easy to understand please change line 51 to https://avatars.githubusercontent.com/rossbb

**Review by Jaretzbalba — DISMISSED** [review]
@trishajjohnson I see the changes made to the roles to keep everything consistent and it looks great! I just left a comment to fix a line concerning the linked image and everything should be good. Thank you again for working on this!

File: _projects/home-unite-us.md (hunk 1 above)
Comment (trishajjohnson) [reply _projects/home-unite-us.md]: Hey @Jaretzbalba I went ahead and made that change you requested for the picture url for Ben Ross.  Would you be so kind as to re-review and approve for me?  Thanks!

**Review by tamara-snyder — APPROVED** [review]
Looks great! The links work and Ben's picture is showing up. Thank you!

**Pull request merged**